diff --git a/node_modules/@pexip/media-processor/dist/index.mjs b/node_modules/@pexip/media-processor/dist/index.mjs
index f6d9ed5..6c6b682 100644
--- a/node_modules/@pexip/media-processor/dist/index.mjs
+++ b/node_modules/@pexip/media-processor/dist/index.mjs
@@ -1097,7 +1097,7 @@ var SEG_MODELS = [
 ];
 
 // src/video/utils.ts
-import { Tensor, browser } from "@tensorflow/tfjs-core/dist/base";
+import { Tensor, browser } from "@tensorflow/tfjs-core";
 var createCanvas = (width, height) => {
   const canvas = document.createElement("canvas");
   canvas.width = width;
@@ -1464,7 +1464,7 @@ var createFrameCallbackRequest = (callback, frameRate, {
 };
 
 // src/video/canvasRenderUtils.ts
-import { Tensor as Tensor2, browser as browser2 } from "@tensorflow/tfjs-core/dist/base";
+import { Tensor as Tensor2, browser as browser2 } from "@tensorflow/tfjs-core";
 var createCanvasRenderUtils = (processingWidth, processingHeight) => {
   const props = {};
   const getImage = (imageName) => {
